### ISSUE TITLE
Quoting keywords/tables might have gone too far

### ISSUE THREAD

**Issue by brudil** [stephenh/joist-ts#816]
I'm wondering if the changes within #810 might be clashing with some quoting that Knex performs automatically [ref 1](https://github.com/knex/knex/issues/2084) [ref 2](https://github.com/knex/knex/issues/1761).

We've got a table/entity abbreviated to `or` which is appearing to be tripple quoted within the final sql query now, having upgraded from 1.102 to latest.

```sql
left outer join "table_name" as "or_s0" on """or"""."id" = "or_s0"."id"
                                           ^^^^^^^^
```

Happy to PR this if we're happy to just remove this quoting from the col1/col2 parts for joins, which is the only place I've seen this happening. And afaik not covered by the tests.

Not entirely au fait with the knex behavior around quoting as it builds the sql.

**Comment by brudil** [comment]
Repro within #817

**Comment by stephenh** [comment]
I went ahead and released #817 with a fix, but let me know if there's anything else!
